simtools/corsika_simtel/corsika_simtel_runner.py

```diff
@@ -0,0 +1,202 @@
+import stat
+from pathlib import Path
+
+from simtools.corsika.corsika_runner import CorsikaRunner
+from simtools.simtel.simtel_runner_array import SimtelRunnerArray
+
+__all__ = ["CorsikaSimtelRunner"]
```

Comment:
I would add here to the list all classes present in the file including the error classes. Otherwise you are not able to import them in the tests later I think. Please add unit tests for these error as well

Reply:
I actually realised I don't need that error class, it stayed from the original class I copied from, so I removed it which solves that problem.

Generally we don't have to list stuff in `__all__`, it is there just in case a user imports using `*` which is really discouraged. For the tests it's not an issue.

Reply:
ok that we dont need.

Your second statement is probably true but there exception for cases in which the function is in the __init__.py file (I am having this problem now in the other PR). nevermind, all good then.